common/src/main/java/com/wynntils/core/net/UrlManager.java

```diff
@@ -64,11 +97,18 @@ public final class UrlManager extends Manager {
     // -Dwynntils.url.list.ignore.cache=true
     private static final String URLS_IGNORE_CACHE_ENV = "wynntils.url.list.ignore.cache";
 
+    // -Dwynntils.url.list.debug.log=true
+    private static final String URLS_DEBUG_LOG_ENV = "wynntils.url.list.debug.log";
```

Comment:
At some point we might want to collect all these system properties in a single place, if you expect to add more like these. Perhaps a SystemPropertiesManager or such. If for nothing else, just to have them all collected in one place and not scattered all over. Otherwiese any name standard is sure to get lost over time.

Reply:
Good idea, I'll make sure to do it sometime.